python/sglang/srt/layers/attention/utils.py

```diff
@@ -53,6 +58,7 @@ def create_flashmla_kv_indices_triton(
     PAGED_SIZE: tl.constexpr = 64,
 ):
     BLOCK_SIZE: tl.constexpr = 4096
+    # Keep in sync with module-level TRITON_PAD_NUM_PAGE_PER_BLOCK constant above
```

Comment:
instead of manually keep in sync, what about 

```
@triton.jit
def create_flashmla_kv_indices_triton(
  NUM_PAGE_PER_BLOCK: tl.constexpr
)

create_flashmla_kv_indices_triton(
  NUM_PAGE_PER_BLOCK=TRITON_PAD_NUM_PAGE_PER_BLOCK,
)
```

alternatively, iirc we are able to directly use global consts inside a triton kernel with some grammar (forget details though)